content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -111,19 +119,53 @@ result.forEach((x) => {
 
 If the database provider is MySQL, the values are `1` or `0`. However, if the database provider is PostgreSQL, the values are `true`, `false`, or `NULL`.
 
-> **Note**: The Prisma Client query engine standardizes the return type for all databases. Using the raw queries does not.
+> **Note**: The Prisma Client query engine standardizes the return type for all databases. **Using the raw queries does not**.
+
+## <inlinecode>$queryRawUnsafe</inlinecode>
+
+The `$queryRawUnsafe` method allows you to pass a raw string (or template string) to the database.
+
+<Admonition type="warning">
+
+By using this method you open up the possibility for SQL injection attacks. SQL injection attacks can expose your data, be it confidential or otherwise sensitive, to being deleted, modified, or even destroyed.
+
+We strongly advice that you use the `$queryRaw` query instead. For more information on SQL injection attacks, see the [OWASP SQL Injection guide](https://www.owasp.org/index.php/SQL_Injection).
```

Comment:
(Maybe that's my english 😄)
Don't we say "We strongly advise" with an 's'?

Reply:
AHHHHHHHH yes we do! I shall take your advice and change it!

![ba-dum-tis](https://user-images.githubusercontent.com/22930449/130211328-a7a6aed7-4bb7-46ba-9f5c-55dfed3745b8.gif)